File: skills/astra-toolkit/clients/java/examples/admin-drop-keyspace.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.DatabaseAdmin;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN")).getDatabase(System.getenv("ASTRA_DB_API_ENDPOINT"));

    // Get an admin object
    DatabaseAdmin admin = database.getDatabaseAdmin();

    // Drop a keyspace
    admin.dropKeyspace("**KEYSPACE_NAME**");
  }
}

File: skills/astra-toolkit/clients/typescript/examples/admin-drop-keyspace.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db(process.env.ASTRA_DB_API_ENDPOINT!, {
  token: process.env.ASTRA_DB_APPLICATION_TOKEN!,
});

// Get an admin object
const admin = database.admin();

// Drop a keyspace
(async function () {
  await admin.dropKeyspace("**KEYSPACE_NAME**");
})();
